webapp/tasks.py

```diff
@@ -0,0 +1,85 @@
+import subprocess
+import time
+
+from invoke import task
+
+
+def wait_until_up(url, max_tries=100, delay=0.1):
+    import requests
+
+    try_num = 0
+    while try_num < max_tries:
+        try:
+            requests.get(url)
+            return
+        except requests.exceptions.ConnectionError:
+            time.sleep(delay)
+            try_num += 1
+
+
+@task
+def test_pytest(c):
+    c.run("pytest")
```

Comment:
what is `c` in all of these tasks? `console`? `client`?

Reply:
I have to run, so allow to me RTFM you: http://docs.pyinvoke.org/en/stable/getting-started.html#defining-and-running-task-functions :P
(The answer is "context".)

Reply:
Haha, thank you. I learned a new abbreviation and about invoke task context. :D